package.json

```diff
@@ -34,5 +43,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@types/lodash": "^4.14.173"
```

Comment:
[question] Where is the actual lodash added as dependency? 🤔  (This only adds the type definitions, not actual implementation. If this was C++ this would be similar to adding the headers but not actual impl file)

Reply:
done